src/assets/scss/style.scss

```diff
@@ -1,7 +1,67 @@
 body {
   // background: #f5f5f5;
 }
+<<<<<<< HEAD
 .article__image {
+=======
+
+
+.View {
+	background: url("https://mdbootstrap.com/img/Photos/Others/img (51).jpg")no-repeat center center;
+	background-size: cover;
+}
+@media (min-width: 768px) {
+	.View {
+			overflow: visible;
+			margin-top: -10px;
+	}
+}
+.navbar {
+	z-index: 1;
+}
+.image_input {
+  display: none;
+}
+.View {
+height: 100%;
+}
+/* Pagination */
+.blog-pagination {
+	margin-bottom: 4rem;
+}
+.blog-pagination > .btn {
+	border-radius: 2rem;
+}
+.articles{
+	margin-top: 100px;
+}
+.social-view{
+	margin-top: 35vh;
+}
+.card-comments img {
+	width: 4rem; }
+.article-icons {
+	display: inline-block;
+	}
+	.Card {
+			margin: 0 auto; /* Added */
+						float: none; /* Added */
+						margin-bottom: 10px; /* Added */
+		}
+		input.form-control.no-border,textarea.form-control.no-border {
+					border: 0 !important;
+					box-shadow: none !important;
+	}
+	.in-font {
+			font-size: 40px;
+
+}
+
+.image-input{
+  display: none;
+}
+.article__image{
+>>>>>>> feature(list bookmarks): List Bookmarked articles-list a users bookmarks[Finishes #161382369]feature(list bookmarks): List Bookmarked articles-list a users bookmarks[Finishes #161382369]
```

Comment:
Could you also perfect the rebase and remove this from your code ouput

Reply:
resolved